### PR TITLE
Fix project creation for staff user 3Blades/app-backend#117

### PR DESCRIPTION
**Pull request by jkp85**
Signed-off-by: Jan Kowalski <jankowalskipraca85@gmail.com>

### CODE REVIEW

File: projects/serializers.py [hunk 1]
```diff
@@ -23,7 +23,11 @@ def create(self, validated_data):
         collaborators = validated_data.pop('collaborators', [])
         project = super().create(validated_data)
         request = self.context['request']
-        Collaborator.objects.create(project=project, owner=True, user=request.user)
+        if request.user.is_staff:
```
Comment (jgwerner) [inline projects/serializers.py]: So `is_admin` property equates to super user, and `is_staff` property equates to all other valid system users?

Comment (jkp85) [inline projects/serializers.py]: Yes, exactly.

File: projects/tests/test_views.py [hunk 2]
```diff
@@ -44,6 +44,22 @@ def test_create_project(self):
         self.assertEqual(Project.objects.count(), 1)
         self.assertEqual(Project.objects.get().name, data['name'])
 
+    def test_create_project_with_different_user(self):
+        staff_user = UserFactory(is_staff=True)
```
Comment (jgwerner) [inline projects/tests/test_views.py]: Should a test be required for `is_admin` users as well?

Comment (jkp85) [inline projects/tests/test_views.py]: User that is not staff cannot be admin, therefore is_admin cannot be true if is_staff is false.

**Pull request merged**